core/Piranha/Services/IMediaService.cs

```diff
@@ -22,7 +24,14 @@ public interface IMediaService
         /// </summary>
         /// <param name="folderId">The optional folder id</param>
         /// <returns>The available media</returns>
-        Task<IEnumerable<Media>> GetAllAsync(Guid? folderId = null);
+        Task<IEnumerable<Media>> GetAllByIdAsync(Guid? folderId = null);
```

Comment:
The `GetAll` naming was just chosen because other services have that method, however I agree that it can be changed for clarity. If we're changing the method, maybe `GetByFolderIdAsync` would be better as the current name implies that the parameter has something to do with the media Id?

Reply:
Good point! I'll make this change, too.